src/patternfly/components/Dropdown/examples/dropdown-disabled-example.hbs

```diff
@@ -0,0 +1,5 @@
+{{#> dropdown id="dropdown-example-collapsed" dropdown--IsActionMenu="true" dropdown--HasToggleIcon="true" dropdown-toggle--IsDisabled="true"}}
```

Comment:
Should you update the id to `id="dropdown-disabled-example-collapsed` as it currently shares the same id as the example above and they may cause a11y issues.